File: Scripts/Listener/ICommonListener.cs
// *******************************************
// Company Name:	深圳市晴天互娱科技有限公司
//
// File Name:		ICommonListener.cs
//
// Author Name:		Bridge
//
// Create Time:		2024/09/03 18:56:54
// *******************************************

namespace Bridge.Common
{
	/// <summary>
	/// 
	/// </summary>
	public interface ICommonListener
	{
		void OnResult(int errCode, string errMsg);
	}
}

File: Plugins/Android/com/bridge/common/ICommonListener.java
package com.bridge.common;

/**
 * 通用监听回调
 */
public interface ICommonListener {
    /**
     * 监听回调 - 错误
     * @param errCode 错误码
     * @param errMsg 错误信息
     */
    void onResult(int errCode, String errMsg);
}
